services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from db.models import CinemaHall
+from django.db.models.query import QuerySet
```

Comment:
Order and group your imports correctly to follow PEP-8 here and in the other files as well